int64 sim_id
bool req
---
bool res
